cogs/quantum_pterodactyl/quantum_pterodactyl.py

```diff
@@ -55,8 +55,8 @@ async def list_commands(self, interaction: discord.Interaction):
         for command in commands_list:
             name, description = command.split(" - ")
             embed.add_field(name=name, value=description, inline=False)
-
-        await interaction.response.send_message(embed=embed)
+        
```

Comment:
Erroneous whitespace. We should setup a linter at somepoint, but this shouldnt be breaking atm